ui/page/wallet/single_wallet_main_page.go

```diff
@@ -328,6 +328,10 @@ func (swmp *SingleWalletMasterPage) changeTab(tab string) {
 // displayed.
 // Part of the load.Page interface.
 func (swmp *SingleWalletMasterPage) HandleUserInteractions(gtx C) {
+	if swmp.checkBox.CheckBox.Update(gtx) {
+		swmp.ParentWindow().Reload()
```

Comment:
why we need reload in here? I see we don't have reason to do it

Reply:
same above